tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,10 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+
+    assert len(items) == 0 
```

Comment:
This looks good, we'd expect items to be empty since there are no electronics in the vendor's inventory. 

A way to beef up this test would be also to assert that the items that were originally there are on lines 28-30 are still in the list after running get_by_category()

```
assert item_a in vendor.inventory 
assert item_b in vendor.inventory 
assert item_c in vendor.inventory